MicrosoftTeams.schema.json

```diff
@@ -193,6 +241,11 @@
                         "maxLength": 2048,
                         "pattern": "^[Hh][Tt][Tt][Pp][Ss]://"
                     },
+                    "mobileUrl": { // TODO: should also add this to the configurableTabs settings in js sdk
```

Comment:
we need to add mobile url field to config tabs too. Ideally this might be part of JS SDK code. so need to find the right place to add this field for configurable tabs.

Reply:
When we say 'mobile' what does that actually mean? Does it encompass large devices like iPads? Should this url be platform specific as well? I imagine a iOS tab might want to look different than an android tab?

Reply:
Maybe we could achieve this by passing in another url replacement string for the platform similar to how we deal with themes and the mobile url is not required?

Reply:
I am not aware of this method. Is there any example or some snippet which can help me understand this better? what do you mean by url replacement string?

Reply:
to answer your question mobile platform will cover Ipad, android tab and android , IOS mobile devices. Since all these devices run either ios or android OS react native package can work on them without any issue. The only major difference between them is the size of the device. So a React native package developer can customize their UI based on the device type.

So we have divided the platforms to web, desktop and mobile. The reason for splitting web and desktop despite them being the same code base is primarily because of the capabilities the app can have between a web site and a native installed app on desktop. For now I believe web and desktop will be used together.

Reply:
Do we need separate Android and iOS URIs here?

Reply:
We would probably add the platform in the tab context through the SDK or query param substitution, right?